### PR TITLE
Image copy sse

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test please

File: libfreerdp/codec/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+	endif()
+
+	if(MSVC)
+		if (CODEC_SSE2_SRCS)
+			set_source_files_properties(${CODEC_SSE2_SRCS} PROPERTIES COMPILE_FLAGS "/arch:SSE2" )
+		endif()
+	endif()
+endif()
+if(WITH_NEON)
+	check_symbol_exists("_M_AMD64"     ""  MSVC_ARM64)
```
Comment (mfleisz) [inline libfreerdp/codec/CMakeLists.txt:76]: `_M_AMD64` is only defined for `ARM64EC` targets (ARM64 Emulation Compatible).
Maybe we should check for `_M_ARM64` here (and maybe also `_M_ARM64EC`)?

**Review by mfleisz — COMMENTED** [review]
Minor comment - otherwise lgtm

File: libfreerdp/codec/CMakeLists.txt (hunk 1 above)
Comment (akallabeth) [reply libfreerdp/codec/CMakeLists.txt:76]: will address this in an upcoming pr.

**Pull request merged**